src/Waargonaut/Decode/Error.hs

```diff
@@ -29,7 +29,7 @@ data Err c e
 --
 data DecodeError
   = ConversionFailure Text
-  | KeyDecodeFailed Text
+  | KeyDecodeFailed
```

Comment:
This constructor change is a breaking change, so please release this as waargonaut 0.2